spec/models/automation_request_spec.rb

```diff
@@ -1,5 +1,5 @@
 describe AutomationRequest do
-  let(:admin) { FactoryBot.create(:user, :role => "admin") }
+  let(:admin) { FactoryBot.create(:user_admin, :userid => 'admin') }
```

Comment:
Does the `userid` matter?

Reply:
```User.super_admin``` searches for a user by ```userid```.

Reply:
:cry: @kbrock This looks like a common use case, should we change something in the factory?

Reply:
So the request is to create a super admin if a super admin does not already exist in the test database?

Will `User.seed` work?

Reply:
```User.seed``` does not work here. Got errors like:
```
     Failure/Error: factory_exists?(factory) ? super : class_from_symbol(factory).create!(*args)

     ActiveRecord::RecordInvalid:
       Validation failed: User: Userid is not unique within region 37
```
with 
```
  let(:admin) { User.super_admin }

  before do
    User.seed
    ...
``` 
@kbrock @bdunne Please advise. Can we do the refactoring in a separate PR if necessary?

Reply:
please dont create an admin user with the wrong groups.

`User.seed` is idempotent - so you can call it multiple times and it should always work.